### PR TITLE
Verodin integration

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/8114

### CODE REVIEW

File: Integrations/integration-Verodin.yml [hunk 1]
```diff
@@ -0,0 +1,859 @@
+versionedfields: {}
+commonfields:
+  id: Verodin
+  version: -1
+name: Verodin
+display: Verodin
+category: Forensics & Malware Analysis
```
Comment (aviadl) [inline Integrations/integration-Verodin.yml]: Is this the correct category ?

File: Integrations/integration-Verodin.yml [hunk 2]
```diff
@@ -0,0 +1,859 @@
...
+    - contextPath: Verodin.Nodes.Name
+      description: Node name
+    - contextPath: Verodin.Nodes.SecurityZoneID
+      description: Node security zone id
+    - contextPath: Verodin.Nodes.Version
+      description: Node version
+    - contextPath: Verodin.Nodes.NetMask
+      description: Node net mask
+    - contextPath: Verodin.Nodes.OrganizationID
+      descri
```
Comment (aviadl) [inline Integrations/integration-Verodin.yml]: what is `sims`

File: Integrations/integration-Verodin.yml [hunk 3]
```diff
@@ -0,0 +1,859 @@
...
+                baseUrl + url + (method === 'GET' && method ? encodeToURLQuery(args) : ''),
+                httpParams,
+                params.insecure,
+                params.proxy,
+                false
+            );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request ' + url + ' failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+        }
+        if (res.StatusCode === 204) {
```
Comment (aviadl) [inline Integrations/integration-Verodin.yml]: What is this ?
Dont you want to return a proper error response ?

File: Integrations/integration-Verodin.yml [hunk 4]
```diff
@@ -0,0 +1,859 @@
...
+            "rawJSON": JSON.stringify(incident),
+        }
+    }
+
+    if (command === 'fetch-incidents') {
+        var lastRun = getLastRun();
+        var lastID = lastRun && lastRun.startID ? lastRun.startID : 0;
+        var startID = lastID;
+        currentCommand = commandDictionary['verodin-get-jobs'];
+        var res = sendRequest(replaceInTemplatesAndRemove(currentCommand.url, args), currentCommand.method, args);
```
Comment (aviadl) [inline Integrations/integration-Verodin.yml]: Where are you limiting the fetched jobs according to startID? 
Getting all jobs and filtering after the fact might cause problems on large scales

File: Integrations/integration-Verodin.yml [hunk 5]
```diff
@@ -0,0 +1,859 @@
...
+    var entries = [];
+    if (currentCommand.extended) {
+        for (var j in currentCommand.translator) {
+            var current = currentCommand.translator[j];
+            var entry = {
+                Type: entryTypes.note,
+                Contents: result,
+                ContentsFormat: formats.json,
+            };
+            var filterFunc = current.filter ? current.filter(filterKey, filterValue) : null;
```
Comment (aviadl) [inline Integrations/integration-Verodin.yml]: Dont think you have here filterFunc

File: Integrations/integration-Verodin.yml (hunk 3 above)
Comment (Spivaka) [inline Integrations/integration-Verodin.yml]: 204 has no body. it's not an error...

File: Integrations/integration-Verodin.yml (hunk 4 above)
Comment (Spivaka) [inline Integrations/integration-Verodin.yml]: Their API doesn't allow filtering (at all). Get jobs get all jobs... I've asked them about it, they didn't reply yet

File: Integrations/integration-Verodin.yml (hunk 2 above)
Comment (Spivaka) [inline Integrations/integration-Verodin.yml]: short name for simulations...

**Pull request merged**